### PR TITLE
Wired up BookshelfCollectionsRepository to CollectionsService

### PR DESCRIPTION
**Pull request by allouis**
image

### CODE REVIEW

**Comment by allouis** [comment]
I've done this separately to the bulk of the changes so that we can release them separately, with the thinking that we might not necessarily want to switch the DB backed storage immediately!

File: ghost/core/core/server/services/collections/service.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
     /** @type {CollectionsService} */
     api;
 
     constructor() {
-        const postsRepository = require('./PostsRepository').getInstance();
-        const collectionsRepositoryInMemory = new CollectionsRepositoryInMemory();
         const DomainEvents = require('@tryghost/domain-events');
+        const postsRepository = require('./PostsRepository').getInstance();
+        const models = require('../../models');
+        const collectionsRepositoryInMemory = new BookshelfCollectionsRepository(models.Collection);
```
Comment (naz) [inline ghost/core/core/server/services/collections/service.js:15]: `bookshelfCollectionsRepository` for a variable name now? or just a `collectionsRepository` 😛

Comment (allouis) [reply ghost/core/core/server/services/collections/service.js:15]: Ha yeah 😝

**Comment by naz** [comment]
Gave it a lil spin. Works great! There's no good reason to hold off the merge for this as we are nearing an alpha version and should be working with close to real world state.

**Pull request merged**